Broken by Makefile removal in debops

I tried using this to install debops:

    stderr: make: *** No rule to make target 'install-scripts'.  Stop.

I see that the Makefile is removed in https://github.com/debops/debops/commit/03cb95d26fc832363b639569e8a41037c486db29
I guess it should run the setup.py instead?

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -19,13 +19,11 @@
 
 - name: Install DebOps playbooks and roles
   command: pip install {{ debops_source_path }}
-  when: debops_register_source is defined and
-        debops_register_source.changed
+  when: debops_register_source.changed
 
 - name: Install DebOps playbooks and roles (async)
   command: debops-update
   async: '{{ (60 * 5) | int }}'
   poll: 0
-  when: debops_playbooks_systemwide is defined and
-        debops_playbooks_systemwide
+  when: debops_playbooks_systemwide
 

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -17,12 +17,12 @@
     update: True
   register: debops_register_source
 
-- name: Install DebOps scripts
+- name: Install DebOps playbooks and roles
   command: pip install {{ debops_source_path }}
   when: debops_register_source is defined and
         debops_register_source.changed
 
-- name: Install DebOps playbooks and roles systemw-wide (async)
+- name: Install DebOps playbooks and roles (async)
   command: debops-update
   async: '{{ (60 * 5) | int }}'
   poll: 0

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -6,7 +6,7 @@
     state: 'latest'
     install_recommends: False
   with_flattened:
-    - [ 'git', 'make' ]
+    - [ 'git', 'python-pip' ]
     - debops_packages
 
 - name: Clone main DebOps source code
@@ -18,14 +18,12 @@
   register: debops_register_source
 
 - name: Install DebOps scripts
-  command: make install-scripts
-           chdir={{ debops_source_path }}
+  command: pip install {{ debops_source_path }}
   when: debops_register_source is defined and
         debops_register_source.changed
 
 - name: Install DebOps playbooks and roles systemw-wide (async)
-  command: make install-playbook
-           chdir={{ debops_source_path }}
+  command: debops-update
   async: '{{ (60 * 5) | int }}'
   poll: 0
   when: debops_playbooks_systemwide is defined and

